src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+
+    margin-right: 4px;
+
+    width: 16px;
+    height: 16px;
```

Comment:
This logic was used before (I mean this square) let's create mixin

Reply:
done